### DISCUSSION TITLE
Setting up fields in QField

### DISCUSSION THREAD

**Discussion by Sergo993108** [opengisch/QField#4024] [Q&A]
Hi all.

There is such a question. It is necessary to configure the fields in the layer with points so that when you click on the check box the following fields related to this item will appear.
For example: we click on a point and start filling data for it. Suppose this point describes some object that has a set of characteristics. We want to describe the transport in this point. Initially we can fill nothing and the point will be empty only with a unique increasing id and all other fields will be devolved to NULL or N/A. We select the object "transport" in the field with the object. Then a field appears where we can select the type of transport, where we choose the car. Then we fill in the other fields, which can characterize the car. But here, we can meet the field of additional modifications. And here, if we click on the check box, or swipe the check-slider, we will have below fields in which we can choose the type of modification, whether it is a trailer, spoiler or other things. If we do not activate the field modification, then the following fields do not appear and after the end of editing points just contain the default value of NULL or N/A or remain empty.

If anyone has any ideas how to implement such functionality, I will be tremendously grateful for it.

**Comment by lucienicolier** [comment]
Hi,

Have a look at the QField doc [Configure Conditional Visibility](https://docs.qfield.org/how-to/attributes-form/#configure-conditional-visibility)
